src/main/java/net/sf/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -1188,9 +1185,9 @@ private void italicizeOrBold(XTextCursor position, boolean italicize, int start,
     }
 
     private void removeReferenceMark(String name) throws NoSuchElementException, WrappedTargetException {
-        XReferenceMarksSupplier xSupplier = UnoRuntime.queryInterface(XReferenceMarksSupplier.class, xCurrentComponent);
-        if (xSupplier.getReferenceMarks().hasByName(name)) {
-            Object o = xSupplier.getReferenceMarks().getByName(name);
+        XNameAccess nameAccess = getReferenceMarks();
+        if (nameAccess.hasByName(name)) {
+            Object o = nameAccess.getByName(name);
```

Comment:
Ah okay. yeah, it looked like you changed it. If it is too much effort then leave it as is.